### PR TITLE
Cupcakemania News Site - Anna H

### PR DESCRIPTION
**Pull request by AnnaHellqvist**
I've tested the webpage in Chrome, Edge and on my Iphone 8.

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -2,21 +2,108 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
   </head>
   <body>
+    <!-- Header with logo and navbar -->
     <header>
-      <!-- Some kind of logo and navbar -->
+      <a href="#">
+        <img class="menu" src="./Images/menu_icon.svg" alt="menu image">
+      </a>
+      <div class="logo"></div>
```
Comment (HIPPIEKICK) [inline code/index.html:18]: 🤔

File: code/index.html [hunk 2]
```diff
@@ -2,21 +2,108 @@
...
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <!-- SMALL NEWS SECTION  -->
+    <section class="small-news-container">
+      <div class="grid-1 grid">
+        <img src="./Images/pic1_unsplash.jpg" alt="Cupcake image">
+        <p>Sprinkles will take your cupcakes to the next level.</p>
+        <br>
```
Comment (HIPPIEKICK) [inline code/index.html:57]: The element `<br>` is not semantic. Try to work around it by adding margin/padding to surrounding elements

File: code/index.html [hunk 3]
```diff
@@ -2,21 +2,108 @@
...
+        <a href="#">Read More</a>
+      </div>
+      <div class="grid-2 grid">
```
Comment (HIPPIEKICK) [inline code/index.html:60]: If your divs are not styled differently, you don't need to give them different class names

File: code/style.css [hunk 4]
```diff
@@ -1,4 +1,215 @@
...
+}
+
+.logo {
+  height: 80px;
+  width: 240px;
+  background: url("./Images/logo_cupcake.jpg") no-repeat;
+}
+
+.logo:hover {
+  background: url("./Images/logo_cupcake_color.jpg") no-repeat;
```
Comment (HIPPIEKICK) [inline code/style.css:36]: 🤩

File: code/style.css [hunk 5]
```diff
@@ -1,4 +1,215 @@
...
+}
+
+nav a:hover {
+  background: rgb(88, 77, 247);
+  color: white;
+}
+
+/*** BIG NEWS ***/
+.big-news {
+  background: linear-gradient(to right, #ff9966, #ff5e62);
```
Comment (HIPPIEKICK) [inline code/style.css:64]: This looks super nice!

File: code/style.css [hunk 6]
```diff
@@ -1,4 +1,215 @@
...
+  /*** BIG NEWS ***/
+
+  .big-news p, h2 {
+    padding: 20px;
+  }
+
+  /*** SMALL NEWS ***/
+
+  .small-news-container {
+    grid-template-columns: 1fr 1fr 1fr 1fr;
```
Comment (HIPPIEKICK) [inline code/style.css:211]: This you can shorten with: `grid-template-columns: repeat(4, 1fr);`

**Review by HIPPIEKICK — APPROVED** [review]
Great job Anna, you managed really well with the CSS grid! The page also gives a uniform impression - with the theme and styling very suiting for the topic. Your code is easy to follow, just put some thought in your class names (how can they be descriptive according to guidelines). With that said -  keep up the good work!

File: code/index.html (hunk 1 above)
Comment (AnnaHellqvist) [reply code/index.html:18]: Hi Matilda, the feedback is a little unclear here, but I guess you mean that I should rename my classes?

Comment (HIPPIEKICK) [reply code/index.html:18]: So sorry! I was thinking the image was the cupcake and that this was just an empty `div`. I misunderstood and I should've made a clearer comment than an emoji!

**Pull request closed without merging**